src/dynatrace/opentelemetry/metrics/export/_exporter.py

```diff
@@ -45,17 +50,18 @@ class _DynatraceMetricsExporter(MetricExporter):
     """
 
     def __init__(
-            self,
-            endpoint_url: Optional[str] = None,
-            api_token: Optional[str] = None,
-            prefix: Optional[str] = None,
-            default_dimensions: Optional[Mapping[str, str]] = None,
-            export_dynatrace_metadata: Optional[bool] = False,
-            preferred_temporality: Dict[type, AggregationTemporality] = None,
-            preferred_aggregation: Dict[
-                type, Aggregation
-            ] = None,
+        self,
+        endpoint_url: Optional[str] = None,
+        api_token: Optional[str] = None,
+        prefix: Optional[str] = None,
+        default_dimensions: Optional[Mapping[str, str]] = None,
+        export_dynatrace_metadata: Optional[bool] = False,
+        preferred_temporality: Dict[type, AggregationTemporality] = None,
+        preferred_aggregation: Dict[type, Aggregation] = None,
```

Comment:
should we drop this too? :thinking:
We will always pass `None` to the superclass constructor to get the default aggregation.

Reply:
👍